core/modules/cart/helpers/createShippingInfoData.ts

```diff
@@ -1,6 +1,5 @@
 const createShippingInfoData = (methodsData) => ({
   shippingAddress: {
-    countryId: methodsData.country,
```

Comment:
As I see, you removed `countryId` and replaced it with `country_id`. Is `countryId` used anywhere?

Reply:
@gibkigonzo 
I think it was only used by event `checkout-after-shippingMethodChanged` in `Checkout.js` and `Shipping.ts` in setting the country of `createOrderData` and `createShippingInfoData` Which had been updated to send full shipping details. The current `checkout-before-shippingMethods` already updates the country in the ShippingDetails so that shouldn't need rework.

And in the testing spec of `checkout-after-shippingMethodChanged` since I changed the data, this probably needs to be updated.

Reply:
ok, great, thank you :). Can you please update test cases?